packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -402,15 +419,15 @@ const tapStatusBariOS = async ( driver ) => {
 };
 
 const selectTextFromElement = async ( driver, element ) => {
-	if ( isAndroid() ) {
-		await longPressMiddleOfElement( driver, element, 0 );
-	} else {
-		await doubleTap( driver, element );
-		await driver.waitForElementByXPath(
-			'//XCUIElementTypeMenuItem[@name="Copy"]',
-			wd.asserters.isDisplayed,
-			4000
+	await longPressMiddleOfElement( driver, element );
+
+	// On iOS long-pressing doesn't select the text automatically
+	if ( ! isAndroid() ) {
+		const selectAllElement = await driver.$(
+			'//XCUIElementTypeMenuItem[@name="Select All"]'
 		);
+		await selectAllElement.waitForDisplayed( { timeout: 1000 } );
+		await selectAllElement.click();
```

Comment:
Noting that we have a [`tapSelectAllAboveElement`](https://github.com/WordPress/gutenberg/blob/0dbf082609992c706c7c363228f37578a44db814/packages/react-native-editor/__device-tests__/helpers/utils.js#L340-L348)utility. I suppose we should either re-use it here or remove the utility.

Reply:
Hey @dcalhoun 👋 

I've checked the `tapSelectAllAboveElement` util but after reviewing it and seeing where it was used (disabled tests) I think it would be best to potentially remove them and find another approach due to the usage of fixed coordinates to select the "Select all" option in the context menu.

When investigating that util I ended up refactoring `selectTextFromElement` in https://github.com/WordPress/gutenberg/pull/55420/commits/306c21776263f84aaed6ad78a0b6dc6f75eef216 because for Android it was only working for elements with just one word. Now it will select all text from the element.